wca-guidelines.md

```diff
@@ -207,8 +207,7 @@ To be more informative, each Guideline is classified using one of the following
 - B1b+) [RECOMMENDATION] Blindfolds should be checked by the WCA Delegate before use in the competition.
 - B2d+) [ADDITION] By default, the competitor starts the solve the first time that they start the timer after the judge has indicated that they are ready (see [Regulation B2a](regulations:regulation:B2a)). If they want to check that the timer is in working order, they must (ask and) receive confirmation from the judge each time before starting/resetting the timer during this phase. Penalty for starting and resetting a timer without confirmation from the judge: disqualification of the attempt (DNF).
 - B2f+) [CLARIFICATION] Since the competitor starts the solve by lifting the cover, penalties for starting the timer and for touching the cover while starting the attempt do not apply (see [Regulation B2b](regulations:regulation:B2b) and [Regulation B2c](regulations:regulation:B2c)).
-- B4c+) [ADDITION] If the judge forgot to put a sight blocker, or places it in a way that the competitor's view of the puzzle is not fully blocked, the WCA Delegate should grant an extra attempt.
-- B4c++) [ADDITION] If the WCA Delegate does not suspect that the competitor intentionally violated any WCA Regulations, the original attempt may stand.
+- B4c+) [ADDITION] If the judge forgot to put a sight blocker and the WCA Delegate does not suspect that the competitor intentionally violated any WCA Regulations, the original attempt may stand.
```

Comment:
Should this stay as B4c++ to avoid renumbering confusion?

Reply:
i believe this has been done in the past (and regardless, for cleanliness/backwards-compatibility sake, it's probably better to keep it as B4c++)

Reply:
In general the numbers aren't changed, but it seemed reasonable here since the wording matched with the original B4c+ with just a change in the outcome. I'm fine either way though

Reply:
```suggestion
- B4c++) [ADDITION] If the judge forgot to put a sight blocker and the WCA Delegate does not suspect that the competitor intentionally violated any WCA Regulations, the original attempt may stand.
```